src/_sass/_main_nav.scss

```diff
@@ -0,0 +1,532 @@
+$nav-blue: #4052a4;
+$nav-bottom-shadow: #f5f5f5;
+$mobile-nav-border: #e5e5e5;
+$hover-color: #f9f9f9;
+
+.headroom {
+  will-change: transform;
+  transition: transform 200ms linear;
+}
+
+.headroom--pinned {
+  transform: translateY(0%);
+}
+.headroom--unpinned {
```

Comment:
missing space - I think the convention is 1 space between style declaration, based on what I've seen in the codebase so far.